site.yml

```diff
@@ -23,6 +23,7 @@
 - hosts: all
   roles:
     - docker
+    - { role: weave, when: weave_enabled|bool }
```

Comment:
Yep you're right
We'll need to move flannel out to a toggle ansible role.
I'm personally in favour of getting hidden disabled features in and create smaller follow ups PRs rather than bigger and more difficult to review PRs